scripts/dns_filter_resolver.h.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/source/extensions/filters/udp/dns_filter/dns_filter_resolver.h b/source/extensions/filters/udp/dns_filter/dns_filter_resolver.h
```

Comment:
I'm assuming we no longer need this patch because it was merged in Envoy, right?

Reply:
Yes, we don't need